hack/id_checker.py

```diff
@@ -0,0 +1,151 @@
+"""
+The ID checker is a support script to check if all the trento checks have an unique ID and
+to set new IDs if they don't exist
+
+:author: xarbulu
+:organization: SUSE Linux GmbH
+:contact: xarbulu@suse.com
+
+:since: 2021-09-16
+"""
+
+import os
+import logging
+import random
+import argparse
+
+try:
+    import yaml
+except ModuleNotFoundError:
+    logging.getLogger(__name__).error(
+        "yaml package not found. To install it run `pip install PyYAML`")
+
+
+CHECKS_FOLDER = "../runner/ansible/roles/checks"
+HEXDIGITS = "0123456789ABCDEF"
+ID_LENGTH = 6
+CHECK_ID = "id"
+REQUIRED_FIELDS = ["id", "name", "group", "labels", "description", "remediation", "implementation"]
+
+
+def parse_args():
+    """
+    Parse arguments
+    """
+    parser = argparse.ArgumentParser(description="Trento Ansible checks ID checker")
+    parser.add_argument(
+        "--generate", dest="generate", action="store_true",
+        help="Generate a new ID on the checks that do not have it")
+
+    return parser.parse_args()
+
+
+def get_checks_folder():
+    """
+    Get the checks folder absolute path
+    """
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), CHECKS_FOLDER)
+
+def create_id():
+    """
+    Create ID with specified lenght based on HEX digits
+    """
+    generated_id = "".join([random.choice(HEXDIGITS) for _ in range(ID_LENGTH)])
```

Comment:
Maybe we could use `uuid.uuid4().hex[ID_LENGTH]` instead, and do without the `HEXDIGITS` constant :)

Reply:
Fixed in: https://github.com/trento-project/trento/pull/282/commits/c796f58561bf795d5050ef3c80944da8d284a9e1